apps/backend/tests/integration/api/test_auth_endpoints.py

```
@@ -0,0 +1,266 @@
+"""Integration tests for authentication endpoints."""
+
+import pytest
+from unittest.mock import patch, AsyncMock
```

Ruff (0.11.9)

`unittest.mock.AsyncMock` imported but unused

Remove unused import: `unittest.mock.AsyncMock`

(F401)